src/client/sandbox/event/focus-blur.js

```diff
@@ -108,17 +108,10 @@ export default class FocusBlurSandbox extends SandboxBase {
             this.lastFocusedElement = null;
     }
 
-    _callFocusCallback (callback, el) {
-        //NOTE:in MSEdge event 'selectionchange' doesn't occur immediately (with some delay)
-        //so we should raise it right after 'focus' event
-        if (browserUtils.isIE && browserUtils.version > 11 && el && domUtils.isTextEditableElement(el))
-            this.eventSimulator.selectionchange(el);
-
-        if (typeof callback === 'function')
-            callback();
-    }
 
     _raiseEvent (el, type, callback, withoutHandlers, isAsync, forMouseEvent, preventScrolling) {
+        //We can not use Promise from the es6-promise library because in IE9, IE10 'resolve' method is called from the setTimeout(1)
```

Comment:
For FocusBlurSandbox we cannot use Promise, because in IE9, IE10 resolve did not called immediately.
It called into setTimeout(callback, 1). See sources - https://github.com/jakearchibald/es6-promise/blob/master/lib/es6-promise/asap.js#L76
It damages some test cases with focus, blur - see build (https://travis-ci.org/testcafe-build-bot/testcafe-hammerhead/jobs/84269763)